<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>411</width>
      <height>461</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string notr="true">Clients</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="disconnectClientButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>122</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnect client</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeClientButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>103</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove client</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
